### DISCUSSION TITLE
projection system codes

### DISCUSSION THREAD

**Discussion by qzchen-uw** [uw-ssec/MAWpy#97] [Ideas]
@gracejia513 @Anurag19101996

Pls develop the codes for dealing with the projection systems and then pass it on to @anujsinha3. Below are the instructions: 

1. for distance calculations, as long as the two  points are in the same projection, there is no need to transform the projection systems.

2. however, if they are not in the same projections, transformation is needed. This happens when some addresses are geocoded in a local projection system while others are in a global system such as WGS 84. Or if the points are geocoded in a local projection system and you want to display them in google map, in that case, a projection transformation is also desired.

3. how to do it? Pls try pyproj library and ask chatgpt while coding the conversion and then finalize. 

Let me know if there are any questions.

**Comment by gracejia513** [comment]
I'm working on this now and will discuss with Cynthia.
